hapi-loader/src/main/java/com/arkhn/hapi/loader/KafkaConsumerConfig.java

```diff
@@ -23,24 +23,24 @@
     @Value(value = "${hapi.loader.kafka.groupId}")
     private String groupId;
 
-    public ConsumerFactory<String, String> consumerFactory(String groupId) {
+    public ConsumerFactory<String, KafkaMessage> consumerFactory(String groupId) {
         Map<String, Object> props = new HashMap<>();
         props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapAddress);
         props.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
         props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
-        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
-        props.put(ConsumerConfig.METADATA_MAX_AGE_CONFIG, 30000);
+        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, MessageDeserializer.class);
+        props.put(ConsumerConfig.METADATA_MAX_AGE_CONFIG, 300);
```

Comment:
Isn't this too low? Isn't it expensive to refresh metadata stuff every 300ms?

Reply:
was useless, fixed